cumulus_logger.py

```diff
@@ -0,0 +1,34 @@
+import json
+from datetime import datetime
+
+class CumulusLogger:
+    """
+    Log messages with contextual info needed by Cumulus.
+
+    Arguments:
+        cumulus_message -- required. either a full Cumulus Message or a Cumulus Remote Message
+        context -- an AWS Lambda context dict
+    """
+    def __init__(self, cumulus_message, context):
+        self.event = cumulus_message
+        self.context = context
+
+    def createMessage(self, message):
+        if type(message) is str:
+            message = {
+                "message": message
+            }
+
+        try:
+            message["level"]
+        except KeyError:
+            message["level"] = "info"
```

Comment:
Something we did in the other loggers is create a function for each level, that way the user of the `log()` function doesn't have to know about the levels, they can call a separate function for error, warning, etc. Then you're not dependent on someone knowing the correct string to use, so I think that would be a good idea here.